src/main/java/org/springframework/data/tarantool/core/convert/MappingTarantoolReadConverter.java

```diff
@@ -89,7 +89,7 @@ public MappingTarantoolReadConverter(EntityInstantiators instantiators,
     @Nullable
     @SuppressWarnings("unchecked")
     public <R> R read(Class<R> targetClass, final @Nullable Object source) {
-        if (source == null) {
+        if (source == null || targetClass.equals(void.class)) {
```

Comment:
This is not a final solution and it's better not to convert the result
to cartridge-java at all, only catch if there is an error. 
We can close ticket #68 and open a new ticket about what needs to be done in a more optimized version. Or, of course, we can just remove this commit and wait until we fix it normally

Reply:
Do not get this change. Would you mind elaborating a bit more on why is this needed?

Reply:
This is done because an error will occur if Tarantool returns any value, but springdata will not be able to convert it to `void`. So far this only happens for `callForObject`, but after this commit 731b415524645af5130b35fbc3f392b2d2ebbc85 will be for callForTuple as well. For example, for this test, where update returns an error, because we return the result, but in the interface `void`.

https://github.com/tarantool/cartridge-springdata/blob/01e3e8a4d621fc360ace54371551bdf1cedb4e37/src/test/java/org/springframework/data/tarantool/repository/support/RepositoryIntegrationTest.java#L145

https://github.com/tarantool/cartridge-springdata/blob/01e3e8a4d621fc360ace54371551bdf1cedb4e37/src/test/java/org/springframework/data/tarantool/repository/BookRepository.java#L20-L21

https://github.com/tarantool/cartridge-springdata/blob/01e3e8a4d621fc360ace54371551bdf1cedb4e37/src/test/resources/cartridge/app/roles/api_router.lua#L24-L26

It seems to me that there are two ways:
1) We close ticket #68 and say that if the user himself wrote garbage and his function from Tarantool returns a result, but in java it expects void, then this is an error. In this case, it is worth writing a suitable exception or using an existing one, because now it is difficult to understand what happened.
2) We ignore the result from Tarantool in carg-springdata or in carg-java. Then the API will be more flexible, but if the user wrote garbage, it will affect the perf and something else

Reply:
How can we understand that such a method is wrong on parsing? We don't know if Tarantool will return a value or not. `void` works fine if it returns nil same as returns nothing. I saw this code where nothing had to be returned in user's code and there needed void